src/js/listeners.js

```diff
@@ -0,0 +1,46 @@
+const { weatherDetails, autoComplete } = require('./data-grab');
+const { removeAllChildNodes, removeAccent } = require('./dom-utils');
```

Comment:
Make use of ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)). Use `import` and `export` instead of `require` or `module.exports`.